tck/src/main/java/ee/jakarta/tck/data/core/example/CDIExampleTests.java

```diff
@@ -42,14 +42,14 @@ public static JavaArchive createDeployment() {
     @Inject
     ExampleService service;
     
-    @Assertion(id = "EXAMPLE", strategy = "Deployes a CDI bean to application server, and injects service to be tested")
+    @Assertion(id = "26", strategy = "Deployes a CDI bean to application server, and injects service to be tested")
     public void serviceAlwaysReturnsTrue() {
         assertTrue(service.getMessage(), "Message should have returned true");
     }
     
     @Disabled(value = "Challenge #1")
-    @Assertion(id = "EXAMPLE", strategy = "Uses the disabled annotation to ensure test does not get executed")
+    @Assertion(id = "26", strategy = "Uses the disabled annotation to ensure test does not get executed")
```

Comment:
Why are all of the assertions using id of `26` ?

Reply:
I looked around other TCKs to see how they related assertions made in the test to the documentation/discussions.

In the platform TCK everything is done in line using comments which bloats the test files. 
Others like the CDI TCK use the spec header names: 
`@SpecAssertion(section = BM_DETERMINING_ANNOTATION, id = "aa")`
For other specs, they don't do anything.

My idea was to link assertions back to GitHub issue numbers.  
Which is where most of the discussion happens. 
Not sure if that is a good idea or not, but figured it was a place to start, and can always be changed :)